src/roscompile/util.py

```diff
@@ -3,15 +3,12 @@
 HASH_PATTERN = re.compile('\n#+\n')
 
 def get_ignore_data_helper(basename):
-    #try:
-        fn = 'package://roscompile/data/' + basename + '.ignore'
-        lines = []
-        for s in get(fn).read().split('\n'):
```

Comment:
I'm finding that this change breaks my installation. Looks like we have a versioning problem with urlgrabber.